test/__snapshots__/index.spec.js.snap

```diff
@@ -1701,7 +1701,39 @@ Array [
 
 exports[`should pass on fixtures/samples/es7/contrived.good.js 1`] = `Array []`;
 
-exports[`should pass on fixtures/samples/es7-object-spread/contrived.good.js 1`] = `Array []`;
+exports[`should pass on fixtures/samples/es7-object-spread/contrived.bad.js 1`] = `
+Array [
+  Object {
+    "column": 7,
+    "line": 7,
+    "rule": "no-unused-vars",
```

Comment:
This snapshot should not contain this error. Simply add a `console.log` statement to make use of that variable.